app/uk/gov/hmrc/thirdpartyapplication/domain/models/ApplicationUpdate.scala

```diff
@@ -30,11 +30,14 @@ trait GatekeeperApplicationUpdate extends ApplicationUpdate {
 }
 
 case class ChangeProductionApplicationName(instigator: UserId, timestamp: LocalDateTime, gatekeeperUser: String, newName: String) extends GatekeeperApplicationUpdate
+case class ChangeProductionApplicationPrivacyPolicyLocation(instigator: UserId, timestamp: LocalDateTime, oldLocation: PrivacyPolicyLocation, newLocation: PrivacyPolicyLocation) extends ApplicationUpdate
```

Comment:
oldLocation probably should not be in the "command"

Reply:
agreed